.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
   - VERSION="10.0" TESTS="0" LINT_CHECK="0"
 
   matrix:
-  - LINT_CHECK="1"
+  - LINT_CHECK="1" EXCLUDE="web_gantt_public_holidays"
```

Comment:
was done to prevent lint check from failing during running on JS code (unknown name `gantt` from outer scope), but, speaking from build logs, it seems like it has no effect on the build anyways - I'll revert this part